### ISSUE TITLE
[REQUEST] Please make the servos so they distribute items evenly.

### ISSUE THREAD

**Issue by deprale** [GabrielOlvH/Industrial-Revolution#308]
**Description:**
- I'm trying to make an efficient "super smelter" without using awful amounts of iron like in vanilla with minecart hoppers and whatnot, and using the item pipes mk1 with output servo from a barrel just sends a load of items to just 1 inventory, is there no option to make it distribute them evenly so its more efficient?

**Reason:**
Efficiency.

**Comment by olee** [comment]
Yeah I had the same problem. For example one possibility would be making it possible to reduce the stack size which an output serve takes out of an inventory so that round robin distribution does not just push a full stack to the first furnace.

Right now I'm using this workaround with a hopper to distribute items in a simple way, but it is not really working that well:
![image](https://user-images.githubusercontent.com/6547794/147866478-1a564dbb-1d49-4a05-859e-0710f4298c74.png)

**Comment by mnsnownutt** [comment]
Do you have the servo set to round robin mode?

**Comment by deprale** [comment]
> Do you have the servo set to round robin mode?  That is an option and though it does not work perfectly, it does an okay job.

That's the problem... It's on round robin and only outputs to like 3 furnaces out of a set of 12 lmao, they are distributed as a line |_||_||_||_| like this.

Also I don't even play the game anymore, but thanks for the reply I guess.
